shared/templates/template_OVAL_zipl_bls_entries_option

```diff
@@ -1,43 +1,12 @@
+    {{{ oval_metadata("Ensure " + ARG_NAME + "=" + ARG_VALUE + " option is configured in the 'options' line in /boot/loader/entries/*.conf. Make sure that newly installed kernels will retain this option, it should be configured in /etc/kernel/cmdline as well.") }}}
     <criteria operator="AND">
-      <criterion test_ref="test_bls_{{{ SANITIZED_ARG_NAME }}}_options"
-      comment="Check if {{{ ARG_NAME_VALUE }}} is present in the 'options' line in /boot/loader/entries/*.conf" />
-      <criterion test_ref="test_kernel_update_{{{ SANITIZED_ARG_NAME }}}_option"
-      comment="Make sure that newly installed kernels will retain {{{ ARG_NAME_VALUE }}} option" />
+      {{{- oval_argument_in_file_criterion('/boot/loader/entries/.*.conf', ARG_NAME, ARG_VALUE, 'Linux kernel') }}}
+      {{{- oval_argument_in_file_criterion('/etc/kernel/cmdline', ARG_NAME, ARG_VALUE, 'Linux kernel') }}}
     </criteria>
   </definition>
 
-  <ind:textfilecontent54_test id="test_bls_{{{ SANITIZED_ARG_NAME }}}_options"
-  comment="check for kernel option {{{ ARG_NAME_VALUE }}} for all snippets in /boot/loader/entries"
-  check="all" check_existence="all_exist" version="1">
-    <ind:object object_ref="object_bls_{{{ SANITIZED_ARG_NAME }}}_options" />
-    <ind:state state_ref="state_bls_{{{ SANITIZED_ARG_NAME }}}_option" />
-  </ind:textfilecontent54_test>
-
-  <ind:textfilecontent54_object id="object_bls_{{{ SANITIZED_ARG_NAME }}}_options"
-  version="1">
-    <ind:filepath operation="pattern match">^/boot/loader/entries/.*\.conf$</ind:filepath>
-    <ind:pattern operation="pattern match">^options (.*)$</ind:pattern>
-    <ind:instance datatype="int" operation="greater than or equal">1</ind:instance>
-  </ind:textfilecontent54_object>
-
-  <ind:textfilecontent54_test id="test_kernel_update_{{{ SANITIZED_ARG_NAME }}}_option"
-  comment="Check for option {{{ ARG_NAME_VALUE }}} in /etc/kernel/cmdline"
-  check="all" check_existence="all_exist" version="1">
-    <ind:object object_ref="object_kernel_update_{{{ SANITIZED_ARG_NAME }}}_option" />
-    <ind:state state_ref="state_bls_{{{ SANITIZED_ARG_NAME }}}_option" />
-  </ind:textfilecontent54_test>
-  <ind:textfilecontent54_object id="object_kernel_update_{{{ SANITIZED_ARG_NAME }}}_option"
-  version="1">
-    <ind:filepath>/etc/kernel/cmdline</ind:filepath>
-    <ind:pattern operation="pattern match">^(.*)$</ind:pattern>
-    <ind:instance datatype="int" operation="greater than or equal">1</ind:instance>
-  </ind:textfilecontent54_object>
-
-  <ind:textfilecontent54_state id="state_bls_{{{ SANITIZED_ARG_NAME }}}_option"
-  version="1">
-    <ind:subexpression datatype="string" operation="pattern match">^(?:.*\s)?{{{ ESCAPED_ARG_NAME_VALUE }}}(?:\s.*)?$</ind:subexpression>
-  </ind:textfilecontent54_state>
+  {{{- oval_argument_in_file_test('/boot/loader/entries/.*.conf', ARG_NAME, ARG_VALUE, 'options ') }}}
```

Comment:
The IDs of tests for `/boot/loader/entries/.*.conf` now have 4 underscores.
`ssg-test_zipl_vsyscall_argument_vsyscall_none_argument_in_boot_loader_entries____conf`

This is not a functional problem, but not nice anyway.

Reply:
Changed `re.sub(r"[^\w]", "_", text)` to `re.sub(r"[^\w]+", "_", text)`.